components/dot_menu/dot_menu.jsx

```diff
@@ -255,6 +257,7 @@ export default class DotMenu extends React.PureComponent {
     render() {
         const isSystemMessage = PostUtils.isSystemMessage(this.props.post);
         const isMobile = Utils.isMobile();
+        const showAddReactionMenuItem = this.props.emojiReactionCount >= Constants.EMOJI_REACTIONS_LIMIT;
```

Comment:
Just to make sure I understand this variable and its usage, if number of current reactions is greater than or equal to the defined limit, this will be true and be used below as part of the consideration for not displaying the reaction menu item? If so, I'm wondering if the variable name might need to be changed because to me it reads that if this is true, the menu item should show.

Let me know if I misunderstand! 🙂

Reply:
Hey @deanwhillier  yes you're absolutely correct, I'll quickly fix this and push the changes !